packages/ui-extensions/docs/surfaces/point-of-sale/reference/apis/product-search-api.doc.ts

```diff
@@ -1,4 +1,11 @@
 import {ReferenceEntityTemplateSchema} from '@shopify/generate-docs';
+import {generateCodeBlock} from '../helpers/generateCodeBlock';
+
+const functionality = 'product-search-api';
+const generateCodeBlockForProductSearchApi = (
+  title: string,
+  fileName: string,
+) => generateCodeBlock(title, functionality, fileName);
```

Comment:
Whats the benefit of creating a helper function within the file? I've just been using the underlying helper `generateCodeBlock`

Reply:
A more succinct equivalent would be

```
const generateCodeBlockForProductSearchApi = generateCodeBlock.bind(null, null, 'product-search-api');
```